src/tracing/src/params.rs

```diff
@@ -83,18 +152,38 @@ impl RustType<String> for CloneableEnvFilter {
     }
 }
 
+impl RustType<String> for SerializableDirective {
```

Comment:
Adding a smoke test for these methods too would be great since changing the `Display` impl could then break proto serialization

Reply:
The existing tests should test the display impl I believe!